lib/commands/validate-spec.ts

```diff
@@ -21,9 +22,11 @@ export async function handler(argv: yargs.Arguments): Promise<void> {
       pretty: argv.p
     }
     if (specPath.match(/.*composite.*/gi) !== null) {
-      await validate.validateCompositeSpec(specPath, vOptions)
+      const result = await validate.validateCompositeSpec(specPath, vOptions)
+      return iterator.some(result, v => !v.validityStatus) ? 1 : 0
     } else {
-      await validate.validateSpec(specPath, vOptions)
+      const result = await validate.validateSpec(specPath, vOptions)
+      return result.validityStatus ? 0 : 1
```

Comment:
can you remind me what validityStatus is for?

Reply:
It's a Boolean flag whish is `true` if the spec is valid and `false` otherwise. Why do we have it? I don't know.

Reply:
ok, is checking on results.length for this case not enough? do we need to check on validityStatus?